app/views/Project/Tagging/Sources/utils.tsx

```diff
@@ -55,3 +55,15 @@ export const getFiltersForRequest = (filters: FilterFormType | undefined) => {
     });
     return requestFilters;
 };
+
+export function getValidDateRangeValues(dateRange?: { startDate: string, endDate: string}) {
```

Comment:
Let's also add why this is useful in context with adding the timezone.

Also, do we need this on the new grapql endpoints?

Reply:
@tnagorra I used the logic from the previous filter. Even I don't know why we require a timezone offset here.